### ISSUE TITLE
N-1

### ISSUE THREAD

**Issue by bdfrost** [pivotal-cf/pcf-pipelines#322]
This is not exactly an issue but would it be possible to upgrade to an N-1 system for patches?

---
Thanks for submitting an issue to pcf-pipelines. To help us triage the issue, please fill out the following form as much as you can:

## Issue

[Provide a quick introduction; please include the pcf-pipelines release you are using, which pipelines you are using (install, upgrade, etc), which IAAS, and the version of PCF you are trying to deploy]

## Context

[Add more details about how this issue is affecting your use of the product]

### Expected result

[Describe what you would expect to have resulted from this process]

### Current result

[Describe what you currently experience from this process]

## Steps to Reproduce

[Detail the process in which we can to find and recreate the issue]

## Possible Fix

[Not obligatory, but suggest fixes or reasons for the bug]

## Logs and Screenshots

[If you have logs or screenshots you can share with us, please do!]

**Comment by bdfrost** [comment]
BTW -- these pipelines are are work of art. I do some concourse work but these are really amazing.

**Comment by bdfrost** [comment]
I really mean 'all'.

**Comment by kcboyle** [comment]
I don't think we can programmatically support N-1 upgrades (I would be difficult if possible within concourse). You _should_, however, be able to hard code the version you're using, and do a `fly sp -p` when there were updates. Not as nice from an automated standpoint, but would accomplish the intended goal of using most recent-1 releases,.
